From aa4e52919264fab0eaba436d5319f5b5b6bf2955 Mon Sep 17 00:00:00 2001
From: Wright Feng <wright.feng@cypress.com>
Date: Wed, 5 Aug 2020 21:07:09 -0500
Subject: [PATCH 3/4] Revert "wifi: disable two debugging flags because not
 support currently"

This reverts commit 78765977b0bdc1deb71a8c7244cc8a261a24f543.

JIRA: SWWLAN-139504
---
 wifi/1.3/default/hidl_struct_util.cpp | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/wifi/1.3/default/hidl_struct_util.cpp b/wifi/1.3/default/hidl_struct_util.cpp
index 0b61c8916..d305c0997 100644
--- a/wifi/1.3/default/hidl_struct_util.cpp
+++ b/wifi/1.3/default/hidl_struct_util.cpp
@@ -157,9 +157,8 @@ bool convertLegacyFeaturesToHidlChipCapabilities(
 
     // There are no flags for these 3 in the legacy feature set. Adding them to
     // the set because all the current devices support it.
-    /* Cypress FMAC does not support those two features, will have them soon */
-    //*hidl_caps |= HidlChipCaps::DEBUG_RING_BUFFER_VENDOR_DATA;
-    //*hidl_caps |= HidlChipCaps::DEBUG_HOST_WAKE_REASON_STATS;
+    *hidl_caps |= HidlChipCaps::DEBUG_RING_BUFFER_VENDOR_DATA;
+    *hidl_caps |= HidlChipCaps::DEBUG_HOST_WAKE_REASON_STATS;
     *hidl_caps |= HidlChipCaps::DEBUG_ERROR_ALERTS;
     return true;
 }
-- 
2.25.0

